src/apis/summary.js

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export default axios.create({
-  baseURL: `${process.env.REACT_APP_API_URL}/v1/`
+  baseURL: `${process.env.REACT_APP_DEV_API_URL}/v1/`
```

Comment:
Why do you seek to distinguish between a dev api and a production one at the code level?